test/client.test.js

```diff
@@ -412,6 +417,115 @@ describe('Client', () => {
       requiredKeys.every(key => driverEnvHeader.includes(key))
     ).toBeDefined()
   })
+
+  test('http2SessionIdleTime env overrides client config', async () => {
+    var client
+    var internalIdleTime
+    const maxIdleTime = 5000
+    const defaultIdleTime = 500
+
+    process.env.FAUNADB_HTTP2_SESSION_IDLE_TIME = '999'
+    client = util.getClient({
+      http2SessionIdleTime: 2500,
+    })
+    internalIdleTime = client._http._adapter._http2SessionIdleTime
+    expect(internalIdleTime).toBe(999)
```

Comment:
To me, it's weird that explicit client specification of `http2SessionIdeltTime` would be superceded by an environment variable. IMHO, the environment variable should be use to provide a customer-specified default.

Reply:
this is the behavior today; so henry is maintaining the current behavior. I agree with you though that's its a bit odd.

https://github.com/fauna/faunadb-js/blob/v4/src/Client.js#L164-L165

https://github.com/fauna/faunadb-js/blob/v4/src/Client.js#L387

Reply:
Yeah I definitely agree as well it is odd but changing that behavior here may be a bit out of scope